### PR TITLE
Flump with FZip example

### PR DESCRIPTION
**Pull request by SmilyOrg**
Depends on a few other things, so probably won't build without them (namely ML-LoomBinaryAsset)

### CODE REVIEW

**Comment by SmilyOrg** [comment]
It's green! Probably needs a review though.

File: docs/examples/FlumpExample/src/FlumpExample.ls [hunk 1]
```diff
@@ -0,0 +1,114 @@
...
+    import loom2d.display.Sprite;
+    import loom2d.display.StageScaleMode;
+    import loom2d.events.Event;
+    import loom2d.Loom2D;
+    import loom2d.textures.Texture;
+    import loom2d.ui.SimpleLabel;
+    import system.platform.File;
+    import system.Void;
+    
+    public class FlumpTest extends Application
```
Comment (bengarney) [inline docs/examples/FlumpExample/src/FlumpExample.ls]: Class doc?

File: docs/examples/FlumpExample/src/FlumpExample.ls [hunk 2]
```diff
@@ -0,0 +1,114 @@
...
+            label.x = stage.stageWidth - 640;
+            label.y = 30;
+            stage.addChild(label);
+            
+            display.x = 100;
+            display.y = 320;
+            display.scale = 0.6;
+            stage.addChild(display);
+            
+            //const loader:Future = LibraryLoader.fromBytes(File.loadBinaryFile("assets/mascot.zip"));
```
Comment (bengarney) [inline docs/examples/FlumpExample/src/FlumpExample.ls]: Dead code?

File: docs/examples/FlumpExample/src/FlumpExample.ls [hunk 3]
```diff
@@ -0,0 +1,114 @@
...
+            const loader:Future = LibraryLoader.fromURL("assets/mascot.zip");
+            loader.succeeded += onLibraryLoaded;
+            loader.failed += function (e:Object) :void { Debug.assert(false, ""+e); };
+        }
+        
+        private function onLibraryLoaded(lib:Object) {
```
Comment (bengarney) [inline docs/examples/FlumpExample/src/FlumpExample.ls]: Can you explain this with comments a bit?

File: docs/examples/FlumpExample/src/FlumpExample.ls [hunk 4]
```diff
@@ -0,0 +1,114 @@
...
+                    manim.push(m);
+                }
+                movies.push(manim);
+            }
+        }
+        
+        override public function onTick() {
+            var dt:Number = 1/60;
+            var spread = 2;
+            for (var mi:int = 0; mi < movies.length; mi++) {
```
Comment (bengarney) [inline docs/examples/FlumpExample/src/FlumpExample.ls]: Can you also explain this a bit?

File: docs/examples/FlumpExample/src/deng/fzip/FZip.ls [hunk 5]
```diff
@@ -0,0 +1,633 @@
...
+		 * Begins downloading the ZIP archive specified by the request
+		 * parameter.
+		 * 
+		 * @param request A URLRequest object specifying the URL of a ZIP archive
+		 * to download. 
+		 * If the value of this parameter or the URLRequest.url property 
+		 * of the URLRequest object passed are null, Flash Player throws 
+		 * a null pointer error.
+		 */		
+        /*
```
Comment (bengarney) [inline docs/examples/FlumpExample/src/deng/fzip/FZip.ls]: Dead code?

**Comment by bengarney** [comment]
How fast is the unzip?

**Comment by SmilyOrg** [comment]
The example spends ~260ms in `_zip.loadBytes` for me.

**Pull request merged**